tests/core/test_flyer.py

```diff
@@ -95,7 +100,7 @@ async def collect_stream_docs(
                     parameters={
                         "path": "",
                         "dataset": "",
-                        "multiplier": False,
+                        "frames_per_event": self._frames_per_event,
```

Comment:
If we are guaranteeing that the first dimension in `shape` is `frames_per_event` I think we won't need it in the parameters anymore.